pydax/_dataset.py

```diff
@@ -172,17 +172,21 @@ def load(self,
                                                              path=self._data_dir / subdataset_schema['path'],
                                                              format_loader_map=format_loader_map)
                 except FileNotFoundError as e:
+                    self._data = None
```

Comment:
Also add an assertion for this in `test_loading_undownloaded`?

Reply:
did you mean this https://github.com/CODAIT/pydax/pull/149/commits/29b2c768789f70bab102ab0f708919181b922a10#diff-26199baaf391361625bd6773b9df4ed2677df4680a9650d2731510adec4898eaR177 or something else?

Reply:
I think I missed that line for some reason. Good work!